src/right-field/results-table/ResultsTable.jsx

```diff
@@ -100,77 +131,19 @@ function generateColumn(variable, size){
  * @param {Function} adder a function which handles what happens with every variable result  
```

Comment:
This parameter is not present in the signature of the function.

Reply:
I have removed this line in [commit dc42dd6](https://github.com/SolidLabResearch/generic-data-viewer/pull/14/commits/dc42dd65bff6c54ddc6d7df1ff67c6529ca74e48)